app/presenters/homepage_presenter.rb

```diff
@@ -0,0 +1,9 @@
+class HomepagePresenter
+  def last_books
+    Book.last(Book::CAROUSEL_LIMIT)
+  end
+
+  def best_sellers
+    BestSellersService.new.call
```

Comment:
It look more as query object, not like a service
incorrect naming
should be `Books::Queries::BestSellers`
or 
`Queries::Books::BestSallers`